tests/openwrt/test_status.sh

```diff
@@ -9,11 +9,14 @@ if [ -z "$1" ] ; then
     exit 1
 fi
 
+LOG_DIR="../../logs"
```

Comment:
Could we use 
```suggestion
LOG_DIR="../../logs/${TARGET}"
```
instead? And then remove the TARGET prefix from the individual files, of course.

Reply:
We can, done.